src/config/migrateConfigOptions.js

```diff
@@ -0,0 +1,19 @@
+const semver = require('semver');
+
+const configOptionMigrations = require('./configOptionMigrations');
+
+function migrateConfigOptions(name, options, fromVersion, toVersion) {
+  if (fromVersion === toVersion) {
+    return options;
+  }
+
+  return Object.keys(configOptionMigrations)
+    .filter((version) => (semver.gt(version, fromVersion) && semver.lte(version, toVersion)))
+    .sort()
```

Comment:
we need to pass semver.compare function here.

Reply:
I still don't understand how I can inspect the object/array when it is being processed in sequence like this. The output of each step in the function is not visible in the debugger, all I can see is the initial object passed to `Object.keys`. I can't see the how the object is transformed after each step. What is the right way to do this?
![image](https://user-images.githubusercontent.com/32928115/102562951-0bf36900-408d-11eb-99a5-5ca2cfec899f.png)

Reply:
I usually check what arguments I'm getting in each function so you can figure out what they return

Reply:
Thanks, I finally understand how this works now :+1: